doc/changelog.rst

```diff
@@ -10,6 +10,8 @@ Version 0.5
 
 **Changes**:
 
+* Add test for SRTomo class.
```

Comment:
Please be more detailed, for example:

    Implement more unit tests for the ``fatiando.seismic.srtomo`` module. Reached 100% test coverage.